components/data-display/week-calendar.tsx

```diff
@@ -100,16 +108,33 @@ export const WeekCalendar: React.FC<WeekCalendarProps> = ({
             カレンダー
           </Heading>
           <HStack>
-            <Button colorScheme="riverBlue" onClick={previousWeek}>
-              前の週
-            </Button>
-            <Button colorScheme="riverBlue" onClick={nextWeek}>
-              次の週
+            {/* 今日の週に戻るボタン */}
+            <Button colorScheme="riverBlue" onClick={goToToday}>
+              今日
             </Button>
+            {/* 前後週の切り替えボタン */}
+            <ButtonGroup>
+              <IconButton
+                icon={<ChevronLeftIcon fontSize="2xl" />}
+                borderTopRightRadius={0}
+                borderBottomRightRadius={0}
+                colorScheme="riverBlue"
+                onClick={previousWeek}
+              />
+              <IconButton
+                icon={<ChevronRightIcon fontSize="2xl" />}
+                borderTopLeftRadius={0}
+                borderBottomLeftRadius={0}
+                colorScheme="riverBlue"
+                onClick={nextWeek}
+              />
+            </ButtonGroup>
```

Comment:
`attached`だけで同じことができるので修正お願いします。

```suggestion
            <ButtonGroup attached>
              <IconButton
                icon={<ChevronLeftIcon fontSize="2xl" />}
                colorScheme="riverBlue"
                onClick={previousWeek}
              />
              <IconButton
                icon={<ChevronRightIcon fontSize="2xl" />}
                colorScheme="riverBlue"
                onClick={nextWeek}
              />
            </ButtonGroup>
```